src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -376,10 +381,30 @@ export default class SpecialResolutionForm extends Mixins(DateMixin) {
       ...this.getcreateResolution,
       signingPerson: this.signingPerson
     })
+    await this.$nextTick()
```

Comment:
This looks like a duplicate when there is already `await this.$nextTick()` inside validate().

Reply:
without this here it is not reflecting on the change of this object, this will happen if you enter one character in ```signingPerson``` object.
I hope there is 2 tick happen, since its updating on watch

Reply:
I see that validate() already waits for async stuff to happen. You probably just need to await validate here (and delete the extra await nextTick).

Reply:
Seems like on the first tick it updates the values and reflect `formValid` current value for all the components, So we need nextTick on validating function.

Reply:
What you have here is TWO await-nextTick: one here and one in validate().

Did you try awaiting validate() and deleting this line?

Reply:
Faiz said he tried this and it didn't work. Can this conversation be resolved? There's another conversation discussing this anyway.

Reply:
I'll resolve this conversation as we can carry on the nextTick discussion below.